lib/report_formatter/chart_common.rb

```diff
@@ -156,6 +156,7 @@ def build_performance_chart_pie(_maxcols, divider)
       return no_records_found_chart if series.empty?
 
       add_axis_category_text(categories)
+      series.zip categories { |ser, category| ser[:tooltip] = category }
```

Comment:
**zip(*args) public**
Converts any arguments to arrays, then merges elements of self with corresponding elements from each argument. This generates a sequence of self.size n-element arrays, where n is one more that the count of arguments. If the size of any argument is less than enumObj.size, nil values are supplied. If a block is given, it is invoked for each output array, otherwise an array of arrays is returned.

Why is the 2nd argument to `zip` a block here?